### ISSUE TITLE
Mobile - Newsletter and contact us submit text not centered on its button on ipad air

### ISSUE THREAD

**Issue by ermatitan** [titan-cebu/Residential-Building-Brokers#13]
![image](https://cloud.githubusercontent.com/assets/12791909/11433177/a28e6724-94f4-11e5-97a5-fbeb00e3dfc4.png)

**Comment by titanguest** [comment]
This is per Bernard's request, as it is need to be raised to Jolene for confirmation.

**Comment by ermatitan** [comment]
A confirmation is needed before we can close the issue. Thanks

**Comment by ermatitan** [comment]
Won't fix issue as confirmed by Bernard Salas
